models/bamf_nnunet_mr_breast/meta.json

```diff
@@ -0,0 +1,134 @@
+{
+    "id": "",
+    "name": "bamf_nnunet_mr_breast",
+    "title": "BAMF MR Breast Segmentation (nnU-Net)",
+    "summary": {
+      "description": "An nnU-Net based model to segment breast from MR scans",
+      "inputs": [
+        {
+          "label": "Input Image",
+          "description": "The MR scan of a patient.",
+          "format": "DICOM",
+          "modality": "MR",
+          "bodypartexamined": "BREAST,FGT,BREAST+TUMOR",
```

Comment:
These are not valid labels under BPE (see https://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html#chapter_L), so `BREAST` would be correct here.